sdv/metadata/multi_table.py

```diff
@@ -599,15 +599,13 @@ def detect_from_csvs(self, folder_name, read_csv_parameters=None):
         if not csv_files:
             raise ValueError(f"No CSV files detected in the folder '{folder_name}'.")
 
+        data = {}
+        read_csv_parameters = read_csv_parameters or {}
         for csv_file in csv_files:
             table_name = csv_file.stem
             self.detect_table_from_csv(table_name, str(csv_file), read_csv_parameters)
+            data[csv_file.stem] = pd.read_csv(str(csv_file), **read_csv_parameters)
```

Comment:
Can we just use the `_load_from_csv` util? Also instead of having to load the CSV twice, would it make more sense to load it and then use `detect_from_dataframe` instead?

Reply:
@frances-h Sure. Do you know if the `detect_table_from_csv` method is user facing? Otherwise this implementation removes the only usage of that method, so I would remove it.

Reply:
@amontanez24 thoughts? It's not in the docs but also doesn't have an _ preceding the function name. Maybe leave it for now and come back to it later?

Reply:
Yeah it's a private method, but you'd want to make sure it doesn't cause anything else to fail. I'm not sure if we use it anywhere else or in enterprise at all

Reply:
It's not used in SDV, SDV-Enterprise or SDGym.

Reply:
Then it should be fine to remove